ports/windows/micropython.vcxproj

```diff
@@ -82,6 +83,9 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <ClCompile />
     <Link />
+    <Link>
+      <ImageHasSafeExceptionHandlers>false</ImageHasSafeExceptionHandlers>
```

Comment:
Why is this needed, and why only for Release|Win32 ?

Reply:
My fault. It's just for an irrelevant inspection for code generation in windows. I forgot to remove. Thanks.